cmake/OpenSimMacros.cmake

```diff
@@ -515,9 +515,9 @@ macro(OpenSimFindSwigFileDependencies OSIMSWIGDEP_RETURNVAL
     # Clean up the output, since it's in the form of a makefile
     # (and we just want a list of file paths).
     if(${_successfully_got_dependencies} EQUAL 0) # return code 0 is success.
-        # '^.*:' matches the first line of the makefile (the output file path).
+        # '^.*cxx:' matches the first line of the makefile (the output file path).
         # '\\\\' matches a single \ (escape for CMake, and escape for regex).
-        string(REGEX REPLACE "(^.*:|\\\\\n)" "" ${OSIMSWIGDEP_RETURNVAL}
```

Comment:
We use SWIG to generate a Makefile listing the file dependencies for running SWIG. The first line is the target file, and contains `cxx:`. The expression containing `:` is used to remove the first line of the Makefile. This is fine on UNIX. On Windows, each subsequent line contains a `:` (e.g., `C:\Users\...`). Adding the `cxx:` fixes how the regex converts the Makefile into a list of file dependencies.